src/main/java/com/softserve/academy/spaced/repetition/utils/validators/ValidationConstants.java

```diff
@@ -48,6 +48,14 @@ private ValidationConstants() {
 
     public static final int PASS_MAX_SIZE = 20;
 
+    public static final int PASS_MAX_SIZE_VAL = 60;
```

Comment:
We have max limit for password above `PASS_MAX_SIZE = 20`

Reply:
we need different constant for DB because passwordEncoder  hashes password to 60 characters, and normal password should be 20